### PR TITLE
Fix alert item

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@rleungx Please take a look at this comment. Thanks!

> PD_cluster_lost_connect_tikv_nums this alarm promsql is '(sum (pd_cluster_status{type="store_disconnected_count"}) by (instance)> 0) ' 'and (sum(etcd_server_is_leader) by (instance)> 0)', I found that it will trigger when TIFLASH is down Because the tag here is store_disconnected_count And the description of the alarm in our document only writes TIKV

**Comment by en-jin19** [comment]
@zjj1002 The changes have been made, please check them, thanks!

File: alert-rules.md [hunk 1]
```diff
@@ -191,28 +191,28 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
...
 
 * 报警规则：
 
-    `sum(pd_cluster_status{type="store_down_count"}) > 0`
+    `(sum(pd_cluster_status{type="store_down_count"}) by (instance) > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
 
 * 规则描述：
 
-    PD 长时间（默认配置是 30 分钟）没有收到 TiKV 心跳。
+    PD 长时间（默认配置是 30 分钟）没有收到 TiKV/TiFlash 心跳。
```
Comment (TomShawn) [inline alert-rules.md:202]:
```suggestion
    PD 长时间（默认配置是 30 分钟）没有收到 TiKV/TiFlash 心跳。 
```

File: alert-rules.md [hunk 2]
```diff
@@ -312,7 +312,7 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
 
 * 报警规则：
 
-    `sum(pd_regions_status{type="down_peer_region_count"}) > 0`
+    `(sum(pd_regions_status{type="down-peer-region-count"}) by (instance)  > 0) and (sum(etcd_server_is_leader) by (instance) > 0)`
```
Comment (lilin90) [inline alert-rules.md:315]: @en-jin19 第一个 instance 后面有俩空格，多了一个？

Comment (en-jin19) [reply alert-rules.md:315]: @lilin90 与 @rleungx 确认这里可以与后面的格式一致只写一个空格，但由于有两个空格不会影响执行命令，因此在这里和对齐的[英文 PR](https://github.com/pingcap/docs/pull/7300) 中先统一保留了两个空格

**Pull request merged**